front/package.json

```diff
@@ -165,8 +165,11 @@
     "start": "import-meta-env-prepare -x .env.example && vite",
     "build": "vite build",
     "test": "import-meta-env-prepare -x .env.example && vitest",
+    "lint": "eslint --ext .ts,.tsx,.js,.jsx src",
```

Comment:
another PR with a lint task was merged, please remove this one

Reply:
Done, I just moved it to another line.